src/main/resources/liquibase/workbench_changelog/19_5_0.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="cuenyad" id="v19.5.0-1">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT count(*) FROM permission WHERE name = 'DELETE_GERMPLASM_LIST'
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			Update description of DELETE_GERMPLASM_LIST permission
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			UPDATE PERMISSION SET name = 'DELETE_GERMPLASM_LIST', description = 'Delete lists not owned by user' WHERE (name = 'DELETE_GERMPLASM_LIST');
```

Comment:
@cuenyad just notice that you are updating the name and it is not really needed.